src/ServiceProvider.php

```diff
@@ -298,8 +299,12 @@ public function registerSocialiteDriver()
         $socialite = $this->app->make(SocialiteFactory::class);
         $config = config('butler.sso');
 
+        $provider = $config['fake'] ?? false
+            ? FakeProvider::class
+            : PassportProvider::class;
```

Comment:
Wondering if this should be allowed in production? Or should we only use the fake provider if `$config['fake']` equals true and the environment is local?